packages/react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -950,13 +950,14 @@ class DropdownInternal extends React.Component<IDropdownInternalProps, IDropdown
       return;
     }
 
-    // hasFocus tracks whether the root element has focus so always update the state.
-    this.setState({ hasFocus: false });
-
     if (this.state.isOpen) {
-      // Do not onBlur when the callout is opened
+      // Do not call onBlur or update focus state when the callout is opened
       return;
     }
+
+    // hasFocus tracks whether the root element has focus so always update the state.
```

Comment:
I think this comment is wrong now?

Reply:
Ah, good point! I'll update to say it tracks whether the root element or dropdown has focus